### ISSUE TITLE
[PE-D][Tester B] Possible invalid input for the address with the / character.

### ISSUE THREAD

**Issue by nus-pe-script** [AY2324S2-CS2103T-W13-1/tp#143]
When you use the keyword `a/<address>`, you probably should not be able to add a `/` inside of the address because it should be a reserved character used to mark a prefix for an attribute and, therefore, should probably throw an error ie `can not use / inside attribute names as it is a reserved key-character`. Instead, it is allowed. You might add new keywords in the future so it's nice to have that in mind.

-------------
Labels: `type.FeatureFlaw` `severity.Low`
original: TarinPairor/ped#2

**Comment by marcus-ny** [comment]
Good point to consider. Perhaps in the case of a tag following in the command such as `a/<address> t/<tags>` and if the address has a substring `t/` inside of it, what will the error message be?
